algorithm_config.yaml

```diff
@@ -53,15 +53,26 @@ inputs:
       required: false
       default: all
     - name: limit
-      description: Maximum number of GEDI granule data files to download from the CMR,
-        regardless of the number of granules within the AOI.
+      description: Maximum number of GEDI granules to subset, regardless of the number
+        of granules within the spatio-temporal range.
       required: false
-      default: "1000"
+      default: "100_000"
```

Comment:
Is this wise? I think we want people to intentionally have to raise the limit to run a very large job.

Reply:
Either way we do it, users will run into a problem:

1. We set the limit arbitrarily low: the user may produce a result that does _not_ include all granules in their specified spatio-temporal range, and then wonder what's wrong with the result, questioning the correctness of the algorithm, possibly reporting a bug, or at least "wasting" support time only to find out that we arbitrarily limited the number of granules for them because they didn't specify their own arbitrarily higher limit. This is actually what happened recently, motivating this particular change.
2. We set the limit arbitrarily high (to a value that is effectively "unlimited"): the user _always_ gets all the granules within their specified spatio-temporal range. The only "problem" being that the output file is too big for them to use. They then realize they need to set a limit, then rerun with their chosen limit. No support time required (at least not likely, or if any at all, just telling them to look in the log file to see how many granules were fetched when no limit was set, and gauge from there how low to set the limit).

The latter is far more likely to require little to no support, and if any, only to help them choose an appropriate limit, or to reduce their spatio-temporal range (perhaps by chunking). Further, in this case there certainly will be no questioning the validity of the algorithm itself, which is perhaps even more important. We do not want users to doubt that the algorithm is working properly, particularly when there it is working perfectly well.

However, I can add more verbiage to the description of `limit` indicating that they might produce an output file that is too large to be usable if they don't either set a limit or keep their spatio-temporal range reasonably small.